src/imitation/scripts/train_adversarial.py

```diff
@@ -117,20 +121,43 @@ def train(
         "monitor_return" key). "expert_stats" gives the return value of
         `rollout_stats()` on the expert demonstrations loaded from `rollout_path`.
     """
-    assert os.path.exists(rollout_path)
-    total_timesteps = int(total_timesteps)
+    if gen_batch_size % num_vec != 0:
+        raise ValueError(
+            f"num_vec={num_vec} must evenly divide gen_batch_size={gen_batch_size}."
+        )
 
-    logging.info("Logging to %s", log_dir)
-    logger.configure(log_dir, ["tensorboard", "stdout"])
-    os.makedirs(log_dir, exist_ok=True)
-    sacred_util.build_sacred_symlink(log_dir, _run)
+    allowed_keys = {"shared", "gail", "airl"}
```

Comment:
Why do this here instead of letting Python error out later?

Reply:
Fail fast before parallelization via Ray, where error logs become super annoying.

Reply:
Okay, sounds reasonable.